exdb/views.py

```diff
@@ -27,7 +27,12 @@ def form_valid(self, form):
         form.instance.author = self.request.user
 
         if 'submit' in self.request.POST:
-            if form.instance.subtype.needs_verification:
+            verification = False
+            for s in form.cleaned_data['subtype']:
+                if s.needs_verification:
+                    verification = True
+                    break
```

Comment:
You could compress this into:
verification = any(s.needs_verification for s in form.cleaned_data['subtype'])